### PR TITLE
[#2] Create Spring Boot Service Boilerplate

### LINKED ISSUE #2: Create the backend service project structure
Create a folder in the repository named `service`. In this folder, create a Java project for a Spring microservice using the following technologies

- Java 11
- IntelliJ
- Gradle
- Spring Boot
- Junit 5
- Spring actuator

Acceptance criteria: The user can start the server. Sending a GET request to /health indicates that the server is online. User can run sample unit and integration tests.

### PR DESCRIPTION
**Pull request by sebhani**
You should be able to run the project and access the /actuator/health endpoint. 
./actuator/health status is up

Also, you should be able to run and pass the default JUnit test.
JUnit test passed

closes #2

### CODE REVIEW

File: service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.switchboard.switchboardapp;
```
Comment (felixlapierre) [inline service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java]: Package name is repetitive, maybe `com.switchboard.app`?

File: service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SwitchboardappApplication {
```
Comment (felixlapierre) [inline service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java]: `SwitchboardappApplication` is repetitive, maybe `SwitchboardApplication`?

**Review by felixlapierre — COMMENTED** [review]
Looking good! Only some minor naming stuff

File: service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java (hunk 1 above)
Comment (mohhef) [reply service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java]: Fixed.

File: service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java (hunk 2 above)
Comment (mohhef) [reply service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java]: Fixed.

File: service/switchboardapp/.mvn/wrapper/MavenWrapperDownloader.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
+/*
```
Comment (FGRCL) [inline service/switchboardapp/.mvn/wrapper/MavenWrapperDownloader.java:1]: The issue #2 liste Gradle as the build tool, but I am only seeing maven files here, are we switching to Maven?

File: service/switchboardapp/pom.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-devtools</artifactId>
+			<scope>runtime</scope>
+			<optional>true</optional>
+		</dependency>
+
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
```
Comment (FGRCL) [inline service/switchboardapp/pom.xml:43]: That's a useful library

File: service/switchboardapp/pom.xml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.3.3.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.switchboard</groupId>
+	<artifactId>switchboardapp</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>switchboardapp</name>
+	<description>Demo project for Spring Boot</description>
```
Comment (FGRCL) [inline service/switchboardapp/pom.xml]: We could make this decription slightly more specific

File: service/switchboardapp/.mvn/wrapper/MavenWrapperDownloader.java (hunk 3 above)
Comment (mohhef) [reply service/switchboardapp/.mvn/wrapper/MavenWrapperDownloader.java:1]: Resolved during meeting, we will switch to maven.

**Review by FGRCL — DISMISSED** [review]

**Review by FGRCL — APPROVED** [review]

File: service/switchboardapp/pom.xml (hunk 5 above)
Comment (mohhef) [reply service/switchboardapp/pom.xml]: Fixed

**Review by felixlapierre — APPROVED** [review]

**Pull request merged**